geomesa-tools/conf/application.conf

```diff
@@ -9,6 +9,31 @@ geomesa {
         { name = "geom",     type = "Point",           index = true, srid = 4326, default = true }
       ]
     }
+    "example-csv-temporal-guard" = {
+      attributes = [
+        { name = "fid",      type = "Integer",         index = false }
+        { name = "name",     type = "String",          index = true  }
+        { name = "age",      type = "Integer",         index = false }
+        { name = "lastseen", type = "Date",            index = false, default = true }
+        { name = "geom",     type = "Point",           index = true, srid = 4326, default = true }
+      ]
+      user-data {
+        geomesa.query.interceptors = "org.locationtech.geomesa.index.planning.guard.TemporalQueryGuard"
+        geomesa.guard.temporal.max.duration = "30 day"
+      }
+    }
+    "example-csv-graduated-guard" = {
+      attributes = [
+        { name = "fid",      type = "Integer",         index = false }
+        { name = "name",     type = "String",          index = true  }
+        { name = "age",      type = "Integer",         index = false }
+        { name = "lastseen", type = "Date",            index = false, default = true }
+        { name = "geom",     type = "Point",           index = true, srid = 4326, default = true }
+      ]
+      user-data {
+        geomesa.query.interceptors = "org.locationtech.geomesa.index.planning.guard.GraduatedQueryGuard"
```

Comment:
this is going to fail if you don't have the config file right? is that going to be confusing?

Reply:
Yeah, I don't know what to do about it though...  I mean, having a user-data key-value would get around that....

Reply:
actually... this is a typesafe config file. you could add it in here.

Reply:
I thought about that.  It seemed weird to toss this file into GeoServer.  But yeah, lemme do that.

Reply:
true... but at least it will work with the CLI tools...